### DISCUSSION TITLE
Timer - Start and stop by just a pulse

### DISCUSSION THREAD

**Discussion by onki69** [EdgeTX/edgetx#1038] [General]
Hi,

I have to apoligize but I am probably not an Edge /OTX expert but I am having some problems with the timers.
I was 30 years in sports timing and a timer was usually triggerd by a single pulse at the start and another pulse (from a finish line beam) to stop (not reset).
In the same way I am using my timers on other TX systems.

I can find a way to start a timer by a trigger pulse (momentary switch or throttle signal) but I cannot find a way to just stop this timer using a stop switch (momentary) or even a touch button on the display.

Am I just blind or is it just possible to reset a timer but to stop it?
Is it also possible to log data just while the timer is running?

Thanks
Onki

**Comment by pfeerick** [comment]
It should be possible (but admittedly unwieldy) through a combination of logical switches and special functions. i.e. a logical switch (or series of them) that basically toggles a logical switch on and off, which thereby starts and stops the timer. you would also have that same logical switch be the trigger for logging.

**Reply by onki69** [reply]
Thanks,

I am missing the flipflop feature in EdgeTX logic switches. Is this work in progress or is it available under a different name?

**Reply by pfeerick** [reply]
I don't believe there are any switches missing from OTX. Perhaps you are after the "sticky key" logical switch... basically acts as a toggle?
